langchain/callbacks/mlflow_callback.py

```diff
@@ -117,7 +117,7 @@ class MlflowLogger:
     Parameters:
         name (str): Name of the run.

```

Comment:
while we're here, should these be run_name, experiment_name, and run_tags?

Reply:
I would suggest matching the style of MLflow, which uses the variables run_name and experiment_name : 
https://mlflow.org/docs/latest/python_api/mlflow.html#mlflow.run 

However, MLflow [doesn't use the parameter](https://mlflow.org/docs/latest/search.html?q=run_tags&check_keywords=yes&area=default) run_tags so I suggest keeping that to 'tags'. The documentation explains that they are tags for the run.

It should be important to note that if we change the parameter names, people who build code with the MLflow callback and set those parameters will have breaking code when they update to a newer version.